sdk/appconfiguration/Azure.Data.AppConfiguration/tests/ConfigurationLiveTests.cs

```diff
@@ -1264,5 +1265,26 @@ public async Task ClearReadOnlySettingNotFound()
                 await service.DeleteConfigurationSettingAsync(testSetting.Key, testSetting.Label);
             }
         }
+
+        [Test]
+        public async Task AddSettingDefaultAAD()
+        {
+            var endpoint = Recording.RequireVariableFromEnvironment("APPCONFIGURATION_ENDPOINT_STRING");
+            var credential = Recording.GetCredential(new DefaultAzureCredential());
+            var configurationClient = new ConfigurationClient(new Uri(endpoint), credential, Recording.InstrumentClientOptions(new ConfigurationClientOptions()));
+
+            ConfigurationClient service = InstrumentClient(configurationClient);
```

Comment:
Might be nice to put these in a GetAADClient method, similar to the GetClient method we have for creating a client from connection string.

Reply:
Ok